crates/l2/Makefile

```
@@ -54,10 +50,23 @@ ETHREX_PROOF_COORDINATOR_DEV_MODE?=true
 # Matches the ports used by the blockchain/metrics dir
 L2_PROMETHEUS_METRICS_PORT = 3702
 
-# Local L1
+DEFAULT_BRIDGE_ADDRESS=$$(grep ETHREX_WATCHER_BRIDGE_ADDRESS .env | cut -d= -f2)
+DEFAULT_ON_CHAIN_PROPOSER_ADDRESS=$$(grep ETHREX_COMMITTER_ON_CHAIN_PROPOSER_ADDRESS .env | cut -d= -f2)
+
+PROVER_SRC_FILES := $(shell find prover/src -type f)
+
+# ==============================================================================
+# L1
+# ==============================================================================
+
 init-local-l1: ## 🚀 Initializes an L1 Lambda ethrex Client with Docker (Used with make init)
```

It's confusing to me why `init-l1` is the `cargo` version and `init-local-l1` the Docker one. I'd rename these to `*-l1-docker`
```suggestion
init-l1-docker: ## 🚀 Initializes an L1 Lambda ethrex Client with Docker (Used with make init)
```

---

Done [afdb79f](https://github.com/lambdaclass/ethrex/pull/3870/commits/afdb79f702fbac15f456bb6ed6e8bedaf64eeb77)!